website/src/components/SearchPage/SearchFullUI.tsx

```diff
@@ -303,14 +314,7 @@ export const InnerSearchFullUI = ({
                 >
                     <div className='text-sm text-gray-800 mb-6 justify-between flex md:px-6 items-baseline'>
                         <div className='mt-auto'>
-                            Search returned{' '}
-                            {totalSequences !== undefined
-                                ? totalSequences.toLocaleString()
-                                : oldCount !== null
-                                  ? oldCount.toLocaleString()
-                                  : initialCount.toLocaleString()}{' '}
-                            sequence
-                            {totalSequences === 1 ? '' : 's'}
+                            {buildSequenceCountText(totalSequences, oldCount, initialCount)}
```

Comment:
Here I'm fixing a bug where it would show 1 sequence if old count or initial count was 1.